### PR TITLE
Added new variant in skills model to show skills with percent

### PR DESCRIPTION
**Pull request by saiteja2145**
Added a skill set variant that shows two columns with percent
Desk
Mobile

### CODE REVIEW

File: src/modules/resume/skill-set/variants/column/properties.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        type: propertyTypes.OBJECT,
+        required: true,
+        name: {
+          name: 'Skill',
+          type: propertyTypes.STRING,
+          value: 'Skill 1',
+          required: true
+        },
+        rating: {
+          name: 'rating',
```
Comment (AKAspanion) [inline src/modules/resume/skill-set/variants/column/properties.js:32]: Wouldn't it be nice to name this property percentage or percent?
@saiteja2145 @ManojSatishkumar

File: src/modules/resume/skill-set/variants/column/render.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+              <div
+                style={{
+                  ...styles.percentContainer
+                }}
+              >
+                <div
+                  style={{
+                    ...styles.percent,
+                    backgroundColor: theme.value.color,
+                    width: `${skill.rating.value}%`
```
Comment (AKAspanion) [inline src/modules/resume/skill-set/variants/column/render.js:73]: Use normalized value, What if the value given is >100  or <0?
@saiteja2145

File: src/modules/resume/skill-set/variants/column/properties.js (hunk 1 above)
Comment (saiteja2145) [reply src/modules/resume/skill-set/variants/column/properties.js]: @AKAspanion 
Updated the changes

File: src/modules/resume/skill-set/variants/column/render.js (hunk 2 above)
Comment (saiteja2145) [reply src/modules/resume/skill-set/variants/column/render.js]: @AKAspanion 
Updated the changes

File: src/modules/resume/skill-set/variants/column/properties.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  },
+  skills: {
+    name: 'Skills',
+    type: propertyTypes.ARRAY,
+    value: [
+      {
+        name: 'Skill',
+        type: propertyTypes.OBJECT,
+        required: true,
+        name: {
```
Comment (RisingGeek) [inline src/modules/resume/skill-set/variants/column/properties.js:25]: `name` key is repeated

File: src/modules/resume/skill-set/variants/column/render.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  const styles = useStyles();
+  return (
+    <div style={styles.root}>
+      <h1 style={styles.headlineText}>{headlineText.value}</h1>
+      <hr />
+      <br />
+      <div style={styles.skillsContainer}>
+        {skills.value.map((skill) => {
+          let percent = skill.percent.value;
+          if (percent > 100) {
```
Comment (RisingGeek) [inline src/modules/resume/skill-set/variants/column/render.js:50]: I think this condition is not required because percentage will be between 0 and 100.

Comment (saiteja2145) [reply src/modules/resume/skill-set/variants/column/render.js:50]: We are not sure what the user will type maybe it is good to add an extra check

File: src/modules/resume/skill-set/variants/column/render.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+            percent = 100;
+          } else if (percent < 1) {
+            percent = 0;
+          }
+          return (
+            <div style={styles.skillContainer}>
+              <div
+                style={{
+                  ...styles.skill
```
Comment (venkateshpachigulla) [inline src/modules/resume/skill-set/variants/column/render.js:59]: In all other places adding styles directly, why using spread operator here to apply styles.

File: src/modules/resume/skill-set/variants/column/metadata.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+const metadata = {
+  name: 'Column',
+  description: 'A two Column varient with percent to show your skill level'
```
Comment (venkateshpachigulla) [inline src/modules/resume/skill-set/variants/column/metadata.js:3]: column variant

File: src/modules/resume/skill-set/variants/column/render.js (hunk 4 above)
Comment (RisingGeek) [reply src/modules/resume/skill-set/variants/column/render.js:50]: okay

File: src/modules/resume/skill-set/variants/column/properties.js (hunk 3 above)
Comment (saiteja2145) [reply src/modules/resume/skill-set/variants/column/properties.js]: @RisingGeek 
Done Review Changes

File: src/modules/resume/skill-set/variants/column/render.js (hunk 5 above)
Comment (saiteja2145) [reply src/modules/resume/skill-set/variants/column/render.js]: @venkateshpachigulla 
Done Review changes

File: src/modules/resume/skill-set/variants/column/render.js [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+const Column = ({ theme, headlineText, skills }) => {
+  const styles = useStyles();
+  return (
+    <div style={styles.root}>
+      <h1 style={styles.headlineText}>{headlineText.value}</h1>
+      <hr />
+      <br />
+      <div style={styles.skillsContainer}>
+        {skills.value.map((skill) => {
```
Comment (AKAspanion) [inline src/modules/resume/skill-set/variants/column/render.js:48]: @saiteja2145 Please see
https://github.com/ManojSatishkumar/msk-components/blob/master/src/modules/resume/sample/variants/basic/render.js#L26

File: src/modules/resume/skill-set/variants/column/metadata.js [hunk 8]
```diff
@@ -1,6 +1,6 @@
 const metadata = {
-  name: 'Column',
-  description: 'A two Column varient with percent to show your skill level'
+  name: 'Column ',
```
Comment (AKAspanion) [inline src/modules/resume/skill-set/variants/column/metadata.js:2]: @saiteja2145 Extra space?

File: src/modules/resume/skill-set/variants/column/render.js (hunk 7 above)
Comment (saiteja2145) [reply src/modules/resume/skill-set/variants/column/render.js]: @AKAspanion 
Done review changes

File: src/modules/resume/skill-set/variants/column/metadata.js (hunk 8 above)
Comment (saiteja2145) [reply src/modules/resume/skill-set/variants/column/metadata.js]: @AKAspanion 
Done review changes

**Pull request merged**